CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
 CHANGELOG for molecule
 ======================
+1.0.7.dev1
+----
```

Comment:
if you have ``dev1`` at the end of the version, it means there was an issue with tagging/packaging.